src/libraries/System.CodeDom/src/Microsoft/CSharp/CSharpCodeGenerator.cs

```diff
@@ -15,8 +15,6 @@ namespace Microsoft.CSharp
 {
     internal sealed partial class CSharpCodeGenerator : ICodeCompiler, ICodeGenerator
     {
-        private static readonly char[] s_periodArray = new char[] { '.' };
```

Comment:
Please revert all changes to files except the two System.Net ones. The other files are in projects that build for netstandard2.0 where the single char overload doesn't exist.

Reply:
Done.